src/prefect/tasks/shell.py

```diff
@@ -34,6 +34,9 @@ class ShellTask(prefect.Task):
         - stream_output (bool, optional): specifies whether this task should log
             the output as it occurs. If enabled, `log_stderr` will be ignored as the
             output will have already been logged. defaults to `False`
+        - log_stream_as_info (bool, optional): boolean specifying whether the
+            streamed output should be logged using `INFO` as its log level. Will be
+            ignored if `stream_output` has not been set to `True`; defaults to `False`.
```

Comment:
What if instead of adding a new kwarg, we modify `stream_output` to optionally take a log level to stream logs at? `True` would still correspond to `debug` (or should we move this to `info`? cc @madkinsz since you recently touched this code).

This gives users more control over the log level used and keeps the options more localized (one option to set, not two).

Reply:
Yeah we've already discussed this at https://github.com/PrefectHQ/prefect/issues/3865 and https://github.com/PrefectHQ/prefect/issues/4231#issuecomment-794090520

I think it'd be worth changing the default to `INFO` as well.